packages/graphql-language-service-types/src/index.js.flow

```diff
@@ -272,6 +282,19 @@ export type CompletionItem = {
   deprecationReason?: ?string,
 };
 
+export type CompletionItemBase = {
```

Comment:
there should be just one `CompletionItem` that ideally extends the vscode typescript definition, or mirrors what we have.